conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -44,7 +44,7 @@ install:
     - cmd: set PYTHONUNBUFFERED=1
 
     # Configure the VM.
-    - cmd: conda.exe install -n root --quiet --yes conda-forge::conda-forge-ci-setup=2
+    - cmd: conda.exe install -n root -c conda-forge --quiet --yes conda-forge-ci-setup=2
```

Comment:
@isuruf are you sure about this? The other CI templates use the `::` syntax which AFAIR is different (more strict?) than `-c`

Reply:
but honestly, I cannot recall where I read about this difference...

Reply:
I can do that, ok. Unless you prefer another PR, but it's probably good to make the templates uniform in one place.

Reply:
It was changed in https://github.com/conda-forge/conda-smithy/pull/932

Reply:
ah, here were the "docs": https://github.com/conda/conda/issues/7941#issuecomment-436578876

Travis in `master` still uses the `::` version: https://github.com/conda-forge/conda-smithy/blob/93ae0b4c425301d3d0c7f96c96bacec59ba3df69/conda_smithy/templates/travis.yml.tmpl#L60

maybe we should change that too.

Reply:
and I guess here too:

https://github.com/conda-forge/conda-smithy/blob/93ae0b4c425301d3d0c7f96c96bacec59ba3df69/conda_smithy/templates/run_osx_build.sh.tmpl#L20